documentation/Overview.md

```diff
@@ -0,0 +1,88 @@
+<!--
+Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+-->
+
+# Overview of Compiler Phases
+
+Each phase produces correct output or fatal errors.
+
+## Prescan and Preprocess
+
+**Input:** Fortran source and header files, command line macro definitions
```

Comment:
In the section **Resolve Names**, the description of the **Command:** seems to be truncated.